js_v9/Block.js

```diff
@@ -78,20 +78,13 @@ function Block(fallingLane, color, iter, distFromHex, settled) {
 		if(attached === undefined)
 			attached = false;
 
-		if(this.angle > this.targetAngle) {
-			this.angularVelocity -= angularVelocityConst * MainHex.dt;
-		}
-		else if(this.angle < this.targetAngle) {
-			this.angularVelocity += angularVelocityConst * MainHex.dt;
-		}
-
-		if (Math.abs(this.angle - this.targetAngle + this.angularVelocity) <= Math.abs(this.angularVelocity)) { //do better soon
```

Comment:
I assume "do better soon" means that this solution is inelegant, and needs re-factoring.